### PR TITLE
Bug 1320335: Fix quoting for mysql probes

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
@bparees ptal :)

File: examples/db-templates/mysql-persistent-template.json [hunk 1]
```diff
@@ -114,7 +114,7 @@
                   "initialDelaySeconds": 5,
                   "exec": {
                     "command": [ "/bin/sh", "-i", "-c",
-                      "MYSQL_PWD='$MYSQL_PASSWORD' mysql -h 127.0.0.1 -u $MYSQL_USER -D $MYSQL_DATABASE -e 'SELECT 1'"]
+                      "MYSQL_PWD=\"$MYSQL_PASSWORD\" mysql -h 127.0.0.1 -u $MYSQL_USER -D $MYSQL_DATABASE -e 'SELECT 1'"]
```
Comment (kwoodson) [inline examples/db-templates/mysql-persistent-template.json]: @mfojtik Does this command need to be interactive(-i)?

Comment (mfojtik) [inline examples/db-templates/mysql-persistent-template.json]: @kwoodson yes, `-i` means SCL enable here ;-)

**Comment by bparees** [comment]
lgtm [merge]

**Comment by dinhxuanvu** [comment]
@bparees Would you mind checking what caused the merge to fail? I couldn't see anything on Jenkins report.

**Comment by bparees** [comment]
@dinhxuanvu it passed one test and failed the other (we run everything twice now) which implies one of them hit a flake.  regardless, it's still in the merge queue and will merge assuming it passes tests during the merge execution.

**Comment by bparees** [comment]
@mfojtik I seem to recall seeing bugs for all 3 DB readiness probes.  are there pulls for the mongo and postgres templates coming?

**Comment by mfojtik** [comment]
@bparees mongo and postgresql are not affected :)

**Comment by bparees** [comment]
ok apparently i hallucinated the comment that said mongo and postgres were
also broken.

**Comment by dinhxuanvu** [comment]
@bparees I can confirm what @mfojtik said. Sally ran Mongo json just fine. I did with postgresql and it was fine as well. Thanks.

**Pull request merged**